_inc/jetpack-jitm.js

```diff
@@ -86,6 +86,13 @@ jQuery( document ).ready( function( $ ) {
 		var $el = $( this );
 
 		var message_path = $el.data( 'message-path' );
+		// Check for sub-pages of the Jetpack dashboard.
+		if (
+			'wp:toplevel_page_jetpack:admin_notices' === message_path &&
+			'#/plans' === window.location.hash
```

Comment:
Rather than checking specifically for this fragment, can we come up with a simple way of just including the fragment in the message_path, so that we handle all fragments (I'm specifically thinking for the Settings area, which also has different fragments, and I could see us wanting to know that).

Reply:
Yes indeed. I've actually been working on a different approach entirely, so I can bypass the problems this current PR has (I talked a bit about it in 2898-gh-jpop-issues). Ideally we would handle all those React pages from inside React. I don't have a PR yet as I am still working on it, but hopefully I'll have something to show soon!